Content.Server/Medical/HealthAnalyzerSystem.cs

```diff
@@ -68,8 +70,13 @@ public void UpdateScannedUser(EntityUid uid, EntityUid user, EntityUid? target,
             if (!HasComp<DamageableComponent>(target))
                 return;
 
+            TryComp<TemperatureComponent>(target, out var temp);
+            TryComp<BloodstreamComponent>(target, out var bloodstream);
+
             OpenUserInterface(user, healthAnalyzer);
-            _uiSystem.SendUiMessage(healthAnalyzer.UserInterface, new HealthAnalyzerScannedUserMessage(target));
+
+            _uiSystem.SendUiMessage(healthAnalyzer.UserInterface, new HealthAnalyzerScannedUserMessage(target, temp != null ? temp.CurrentTemperature : 0,
+                bloodstream != null ? bloodstream.BloodSolution.FillFraction : 0 /*bloodstream.BloodSolution.FillFraction * 100*/));
```

Comment:
You shouldn't be modifying this function either, try getting the data where I suggested above.